swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -1426,6 +1426,23 @@ module Exprs {
     }
   }
 
+  /** Control-flow for a Pack Expansion. See the QLDocs for `PackExpansionExpr` and `PackElementExpr` for the semantics. */
```

Comment:
There is no QLDoc on `PackExpansionExpr` nor `PackElementExpr` that explains the semantics of them. Is that a copy/pasting gone wrong? Or did you intend to write down what these actually mean? 😄

Reply:
Yeah, I think it's just a bad copy-paste. There are _some_ docs, but I guess those do not explain semantics. I'll update the docs and add some links to the official docs.